### DISCUSSION TITLE
OpenVINO on Mac

### DISCUSSION THREAD

**Discussion by consecutiotemporum** [rupeshs/fastsdcpu#87] [Q&A]
Hi.
It says in the readme that OpenVINO is not supported for FastSD on Mac, but installing FastSD on my Mac made me download this: "Downloading openvino-2023.2.0-13089-cp310-macosx_11_0_arm64.whl (21.7 MB)"
Thx.

**Comment by rupeshs** [comment]
@consecutiotemporum Please try this https://github.com/rupeshs/fastsdcpu#fastsd-cpu-on-mac
